src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs

```diff
@@ -17,7 +17,8 @@ public class PhoneFlyoutPageRenderer : UIViewController, IPlatformViewHandler
 	{
 		UIView _clickOffView;
 		UIViewController _detailController;
-		VisualElement _element;
+		WeakReference<VisualElement> element;
+		VisualElement _element => element?.GetTargetOrDefault();
```

Comment:
Can you instead do:
```suggestion
		WeakReference<VisualElement> _element;
```
Which also follows the naming guidelines by using `_` prefixed names.

It also makes it more clear how many times `_element.TryGetTarget()` is accessed. This is several times slower than accessing a plain field because it goes through a "GC handle" to find the object.

Reply:
@jonathanpeppers Done. Thanks for the suggestion.